R/boxplot.R

```diff
@@ -0,0 +1,14 @@
+#' Plot a boxplot with the magma color scheme and an option to facet.
+#'
+#' @param df Dataframe containing the variables for plotting
+#' @param x Column name of the numerical variable to view the distribution of
+#' @param y Column name containing the categorical variables to assign boxes to
+#' @param facet Determines whether separate graphs will be created for each category
+#'
+#' @return a ggplot object
+#' @export
+#'
+#' @examples
+#' function(df, "Length", "Species", TRUE)
```

Comment:
```suggestion
#' boxplot(df, Length, Species, TRUE)
```
I deleted the quotations around the column name since in R we don't have to use string versions of variable names. Please correct me if I am mistaken. Thanks!

Reply:
That's true, good spot. I will make the change now.